origin-graphql/src/utils/eventCache.js

```diff
@@ -1,10 +1,32 @@
 import { get } from 'origin-ipfs'
 // import { post } from 'origin-ipfs'
 import uniqBy from 'lodash/uniqBy'
+import chunk from 'lodash/chunk'
+import flattenDeep from 'lodash/flattenDeep'
 import createDebug from 'debug'
 
 const debug = createDebug('event-cache:')
 
+// Given a contract and block range, break the range up into requests of no
+// more than 20,000 blocks, then send those requests in batches of 7.
```

Comment:
nit: mention that after experimentation 7 is the sweet spot. otherwise is seems kinda random :)